api/src/main/java/org/apache/iceberg/expressions/Expressions.java

```diff
@@ -105,6 +105,16 @@ public static Expression not(Expression child) {
     return new UnboundPredicate<>(Expression.Operation.NOT_EQ, ref(name), value);
   }
 
+  public static <T> UnboundPredicate<T> in(String name, T value, T... values) {
```

Comment:
If there is only one value in the set, this should return an `equal` predicate. Similarly, `notIn` should return a `notEqual` predicate for a single value.

Reply:
👌